packages/orchestration/src/exos/chain-hub.js

```diff
@@ -133,19 +153,16 @@ export const makeChainHub = (agoricNames, zone = makeHeapZone()) => {
       // Either from registerChain or memoized remote lookup()
       if (chainInfos.has(chainName)) {
         return /** @type {Vow<ActualChainInfo<K>>} */ (
-          watch(chainInfos.get(chainName))
+          vowTools.asVow(() => chainInfos.get(chainName))
         );
       }
 
-      return watch(E(agoricNames).lookup(CHAIN_KEY, chainName), {
-        onFulfilled: chainInfo => {
-          chainInfos.init(chainName, chainInfo);
-          return chainInfo;
-        },
-        onRejected: _cause => {
-          throw makeError(`chain not found:${chainName}`);
-        },
-      });
+      const lookup = vowTools.retriable(
+        zone,
+        'lookupChainInfo',
+        lookupChainInfo,
+      );
```

Comment:
Sorry I wasn't clear earlier but the `vowTools.retriable` call needs to be done outside the method call, likely where `lookupChainInfo` is currently defined.

Reply:
ah that makes sense